Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.17 as builder
+FROM quay.io/konveyor/builder as builder
```

Comment:
I would use a tagged builder instead of latest.
https://quay.io/repository/konveyor/builder?tab=tags&tag=latest

Reply:
```suggestion
FROM quay.io/konveyor/builder:ubi8-v1.18 as builder
```